downloader/src/test/kotlin/DirectoryTest.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2017 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.downloader
+
+import com.here.ort.model.Package
+import io.kotlintest.Spec
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.matchers.shouldThrow
+
+import io.kotlintest.specs.StringSpec
+
+class DirectoryTest : StringSpec() {
+
+    private val outputDir = createTempDir()
+
+    override fun interceptSpec(context: Spec, spec: () -> Unit) {
+        spec()
+        outputDir.deleteRecursively()
+    }
+
+    init {
+        "Creates directories for gradle submodules" {
+            val pkg = Package(
+                    packageManager = "Gradle",
+                    namespace = "",
+                    name = "project :model",
+                    version = "",
+                    declaredLicenses = sortedSetOf(),
+                    description = "",
+                    homepageUrl = "",
+                    downloadUrl = "",
+                    hash = "",
+                    hashAlgorithm = "",
+                    vcsPath = "",
+                    vcsProvider = "",
+                    vcsUrl = "",
+                    vcsRevision = ""
+            )
+            //no download source specified, we expect exception in this case
```

Comment:
Please write all comments like documentation, with proper spelling and punctuation. (Single space after `//`, start sentence with upper case `N`, end sentence with `.` etc. Also please add a blank line before the comment to make the block stand out.